### ISSUE TITLE
Chat stopped working  1.6.7

### ISSUE THREAD

**Issue by JorellAponte** [AllTheMods/ATM-6#1734]
Every time someone tries to use the chat, the text doesn't appear and an error shows up on the console. The server was working fine after the update but after some time the chat stopped working, I restarted the server and all but it didn't fix it 

[https://pastebin.com/C0z4Prt4](url)

**Comment by Athena-AI** [comment]
I found in the reddit that apparently for it to work you need to do "/say [message]" in order for it to work. I tested it and it worked. I hope this helps.

**Comment by JorellAponte** [comment]
> I found in the reddit that apparently for it to work you need to do "/say [message]" in order for it to work. I tested it and it worked. I hope this helps.

I'm not trying to talk from the console, it's the chat in-game that is broken

**Comment by Athena-AI** [comment]
I know but doing the /say in the chat works in-game too.

**Comment by JorellAponte** [comment]
> I know but doing the /say in the chat works in-game too.

it doesn't let me use that command in game
![image](https://user-images.githubusercontent.com/3671872/120256250-b89c5e00-c25b-11eb-822a-12ca89e2009f.png)

**Comment by Athena-AI** [comment]
Oh I guess its because I was opped that it lets me do it then.

**Comment by JorellAponte** [comment]
> Oh I guess its because I was opped that it lets me do it then.

I made me Op used the command and tried to use chat but I got the same error

**Comment by Athena-AI** [comment]
Odd it was working for me

**Comment by Raidace** [comment]
I have the same issue on v1.6.6 of the modpack. It was working fine and then out of the blue it stopped working.

**Comment by JorellAponte** [comment]
Yeah same here, I don't know what caused it, I even reinstalled all the mods just keeping the world folder and it didn't fix it

**Comment by AgentSanchez** [comment]
I can confirm this issue is happening after updating as well.

**Comment by TiimK** [comment]
Relevant error when chat message is sent.
https://pastebin.com/pEsv5uw8

This is due to an issue directly related to AdvancedPeripherals. It appears the chatbox part is causing issues with chat. Even setting chatbox to false in the config didn't help. Only way to fix this for the time being that I've found is to remove the chatty turtles.

You can either wait for a chunk with the chatty turtle to unload at which point chat will function again or do /advancedperipherals debug

That will give you the position of all the turtles and you can remove them yourself. They only appear to be in villages so tell your players to destroy them when they come across them until a fix is found.

**Comment by Seniorendi** [comment]
I will fix this issue soon, but there is a workaround:

1. Use the command `/advancedperipherals debug`
2. You will receive a list of block locations, try to locate a "chatty turtle"
3. Break the chatty turtle and the chat will work again

**Comment by ladylexxie** [comment]
This is fixed in one of the updates from Advanced Peripherals

**Comment by Rocksnotch** [comment]
> This is fixed in one of the updates from Advanced Peripherals

> I will fix this issue soon, but there is a workaround:
> 
> 1. Use the command `/advancedperipherals debug`
> 2. You will receive a list of block locations, try to locate a "chatty turtle"
> 3. Break the chatty turtle and the chat will work again

not sure if its worth it to bring this up again but a fully up-to-date (as of 6/9/2021) server had a village spawn one of these chatty turtles and it completely stopped chat. So not sure if entirely fixed.

**Comment by Seniorendi** [comment]
> not sure if its worth it to bring this up again but a fully up-to-date (as of 6/9/2021) server had a village spawn one of these chatty turtles and it completely stopped chat. So not sure if entirely fixed.

Yeah, it was not fixed. Fixed in 0.6.6b
